trin-core/src/portalnet/storage.rs

```diff
@@ -700,7 +709,9 @@ impl PortalStorage {
 
     /// Helper function for opening a SQLite connection.
     pub fn setup_sql(node_id: NodeId) -> Result<Pool<SqliteConnectionManager>, ContentStoreError> {
-        let mut data_path: PathBuf = get_data_dir(node_id);
+        let mut data_path: PathBuf = get_data_dir(node_id).map_err(|err| {
+            ContentStoreError::Database(format!("unable to get data dir: {:?}", err))
```

Comment:
```suggestion
            ContentStoreError::Database(format!("unable to get data dir for sql: {:?}", err))
```

for error uniqueness